README.md

```diff
@@ -709,6 +718,21 @@ The response object is visualized as follows:
 
 TODO
 
+### `sourceTVGamesData` (`sourceTVGamesResponse`)
+* `sourceTVGamesResponse` - The raw response object
```

Comment:
Is it normal to have player info in sourceTVGamesData? didn't you switch it up with the playerInfoData event?